.github/ISSUE_TEMPLATE/out_of_date_package_report.md

```diff
@@ -1,45 +1,39 @@
 ---
 name: Out-of-date package reports
 about: For packages that are out-of-date
-title: ''
+title: 'Update request: PACKAGENAME OLDVERSION → NEWVERSION'
 labels: '9.needs: package (update)'
 assignees: ''
 
 ---
 
+##### Package details
 
-###### Checklist
-
-<!-- Note that these are hard requirements -->
-
+- Package name:
```

Comment:
All of this seems redundant, given the package name and versions are already in the title.

I think this whole template needs to be reworked. What about this: Instead of scattering non-linear instructions all over the place, provide a major comment listing all steps to perform to collect the required information and where to put it. There should be no redundancies, because these are opportunities for errors and annoyance.

Reply:
I'm in favor of removing redundancy, however I think the title should only contain redundant information by design.

Reply:
> the title should only contain redundant information by design

Why? You did not address my objection that performing more steps or entering the same information multiple times adds error surface.

Reply:
The title of an issue should summarize an issue's content and thus not provide any new information after having read the issue. This is what I mean with "redundant by design". In this case, the relevant summarizing information are the package name and outdated/fresh versions.

And yes, there's always the possibility to mess up an issue title, but there's also the possibility to edit it later on. Nothing on that is new or specific to this template IMO.